### PR TITLE
Update Parlot to 1.0

### LINKED ISSUE #35: Update Parlot to 1.0.0
YesSQL 5.1.0 depends on Parlot 1.0.0. we can't used YesSQL 5.1.0 in OC along with shortcode unless we update shortcode to also use Parlot 1.0.0+

### PR DESCRIPTION
**Pull request by MikeAlhayek**
Fix #35

### CODE REVIEW

**Comment by MikeAlhayek** [comment]
@sebastienros please approve this workflow to run. 

Also this line is throwing an exception
https://github.com/MikeAlhayek/shortcodes/blob/280261cb668e65f6c77c0d967ad0477bde5b85b7/src/Shortcodes/ShortcodesParser.cs#L146

> System.ArgumentOutOfRangeException : Specified argument was out of the range of valid values. (Parameter 'start')

Maybe using `_scanner.Cursor.Offset` is not the correct value for start.

File: src/Shortcodes/ShortcodesParser.cs [hunk 1]
```diff
@@ -128,17 +128,17 @@ private Shortcode ParseShortcode()
                 {
                     arguments ??= CreateArgumentsDictionary();
 
-                    arguments[argumentIndex.ToString()] = Character.DecodeString(new TextSpan(_scanner.Buffer, _result.Start + 1, _result.Length - 2)).ToString();
```
Comment (sebastienros) [inline src/Shortcodes/ShortcodesParser.cs:131]: This is probably wrong. Start is the beginning of the result, and Offset is the current position. So Offset should be stored as a local Start before the call to Read

**Comment by MikeAlhayek** [comment]
@sebastienros yes that is what I am thinking too. Because the line that is throwing the exception is in Parlot.

**Comment by sebastienros** [comment]
You can close this PR, I had to do several changes in Parlot to get all the tests passing

**Pull request closed without merging**